text/0000-icons.md

```diff
@@ -235,10 +260,16 @@ For example, here is an icon implementation from the Classic design system befor
 
 This import will replace a direct SVG import, and the contents can be assumed to be an SVG element that is directly usable by your framework.
 
+When a developer needs an icon asset for their framework, they would only need to know to come to this package, and what it provides.
+
 # Unresolved questions
 
 >**Optional, but suggested for first drafts. What parts of the design are still TBD?**
 
 How are we sharing icons for other platforms, such as mobile native?
 
 How do we make the icons availables for designers to use?
+
+What is our process for adding new framework/format support?
+
+Should we provide an iterable way to communicate all available options (that doesn't interfere with treeshaking)?  Something that would be useful in automating the build of a demo page and/or providing hints via IDEs.
```

Comment:
Using TS by default will give IDE hints which we'll need to do because our teams (and now the world) basically lives off of it.

Reply:
Ok. I opted for babel instead, since there wasn't much typing and we're using imports for the icons, which I didn't think we'd get any IDE help with from TS - but I could be wrong.  I can add types to the elements and something that defines the existing icon choices, but I don't know how we'd make it usable without breaking tree shaking.  I can look into it and work on that.

Reply:
Babel has TS plugins and presets. 😉